settings.gradle

```diff
@@ -5,6 +5,6 @@ plugins {
 apply plugin: 'com.diffplug.blowdryerSetup'
 
 blowdryerSetup {
-    //github('GTNewHorizons/ExampleMod1.7.10', 'tag', '0.1.4')
-    devLocal '.' // Use this when testing config updates locally
+    github('GTNewHorizons/ExampleMod1.7.10', 'tag', '0.1.4')
```

Comment:
:+1:  You can also kill the `spotless.gradle` since it'll pull from the Example Mod repo